src/platforms/rp2/src/lib/CMakeLists.txt

```diff
@@ -67,6 +66,10 @@ target_link_libraries(
         pico_sync
         pico_time)
 
+if (PICO_RP2040)
+    target_link_libraries(libAtomVM${PLATFORM_LIB_SUFFIX} PUBLIC hardware_rtc)
```

Comment:
Just for my personal understanding, what about RTC on RP2350?

Reply:
RTC is not supported on RP2350. Instead, they feature the standard RISC-V platform timer, and for this PR I didn't port the nif for RTC.

See https://www.raspberrypi.com/documentation/pico-sdk/hardware.html#hardware_riscv_platform_timer-rp2350